pulsar/README.md

```diff
@@ -29,6 +29,25 @@ No additional installation is needed on your server.
 
 See [metadata.csv][7] for a list of metrics provided by this check.
 
+
+### Log collection
+
+1. The Pulsar log integration supports Pulsar's [default log format][10]. Clone and edit the [integration pipeline][11] if you have a different format.
+
+2. Collecting logs is disabled by default in the Datadog Agent. Enable it in your `datadog.yaml` file:
+   ```yaml
+   logs_enabled: true
+   ```
+
+3. Uncomment and edit the logs configuration block in your `pulsar.d/conf.yaml` file. Change the path and service parameter values based on your environment. See the [sample pulsar.d/conf.yaml][4] for all available configuration options.
+   ```yaml
+    logs:
+      - type: file
+        path: /pulsar/logs/pulsar.log
+        source: pulsar
```

Comment:
Looks good! Small nit. Since we're talking about editing the service parameter, should the sample config have a service param as an example?

Reply:
Ah true, I think I can just remove that line about the service param!